ReSharper.FSharp/src/FSharp.Psi.Features/src/CodeCompletion/FSharpScriptReferenceCompletionProvider.fs

```diff
@@ -113,6 +130,9 @@ type FSharpPathCompletionProvider() =
         | "I"    -> Some (Set.empty, null)
         | _ -> None
 
+    let nugetItem =
+        ReferenceLookupItem("nuget: ", "nuget", true).WithRelevance(UInt64.MaxValue).As<TextLookupItemBase>()
```

Comment:
Please allow ignoring `:` and ` ` if user types them manually, like it's done for ` = ` in record fields`